### PR TITLE
De-register api command from Ersilia CLI

### PR DESCRIPTION
**Pull request by adebisi4145**
Thank you for taking your time to contribute to Ersilia, just a few checks before we proceed
- [x] Have you followed the guidelines in our [Contribution Guide](https://github.com/ersilia-os/ersilia/blob/master/CONTRIBUTING.md)
- [ ] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

**Description**

There's a command line `api` in Ersilia that calls different APIs like `run`, `calculate` and so on. Ersilia no longer allow this, so the command is deprecated and needs to be deleted from Ersilia CLI 

**Changes to be made**

1. Run a model to ensure I have a functional installation of Ersilia
2. Create a branch and switch to the branch
3. Remove the `api` command from the `create_cli.py` script
4. Remove the implementation of this command
5. Run a model as before using the `ersilia run` command.
6. Check that the CLI does not crash by running `ersilia api run...` and that there's an error message

**Status**

- I ran a model using `ersilia api run` [here](https://github.com/user-attachments/files/17250759/eos935d.log)
- I created a branch called `DeregisterApi` and switched to it
- I removed the `cmd.api()` command from the `create_cli.py` script [here](https://github.com/adebisi4145/ersilia/commit/dec5c1c6b0de9ce344bba741d1b23de91dabb69a)
- I removed the implementation of this command in [cmd.py](https://github.com/adebisi4145/ersilia/commit/f161065545f035ce6a7409eb51a3a69609a552b8), [api.py](https://github.com/ersilia-os/ersilia/pull/1289/commits/cdd261d790bc50dbcf8e0bef8580d70d9a08a5eb) and deleted api.py file
- I successfuly ran the same model as before using the `ersilia run` command.
- I ran the model again using `ersilia api run...` , the CLI did not crash and I also got an error message [here](https://github.com/user-attachments/files/17250924/error.log)

Is this pull request related to any open issue? If yes, replace issueID below with the issue ID

Related to #1263

### CODE REVIEW

**Comment by adebisi4145** [comment]
I look forward to your feedback @DhanshreeA 
Thank you for your time

**Comment by P-Mefut** [comment]
hi @adebisi4145, if your removing the command, shouldn't the command file `api.py` also be removed?

**Comment by adebisi4145** [comment]
Hello @P-Mefut  , I'm currently working on that. Thank you

**Comment by P-Mefut** [comment]
Well if the command is no longer required then it's best to remove the api.py file... but I prefer we wait on @DhanshreeA to shed more light on it!!!

**Comment by DhanshreeA** [comment]
> Well if the command is no longer required then it's best to remove the api.py file... but I prefer we wait on @DhanshreeA to shed more light on it!!!

Exactly, thanks @P-Mefut. Please take care of this @adebisi4145 :) Thank you for your inputs, both.

File: ersilia/cli/commands/api.py [hunk 1]
```diff
@@ -1,49 +0,0 @@
-import click
```
Comment (DhanshreeA) [inline ersilia/cli/commands/api.py:1]: We should remove the file entirely.

**Review by DhanshreeA — CHANGES_REQUESTED** [review]

**Comment by adebisi4145** [comment]
> Exactly, thanks @P-Mefut. Please take care of this @adebisi4145 :) Thank you for your inputs, both.

Duly note @DhanshreeA , working on that immediately

File: ersilia/cli/commands/api.py (hunk 1 above)
Comment (adebisi4145) [reply ersilia/cli/commands/api.py:1]: Thank you @DhanshreeA  for the feedback, This has now been done

**Comment by Malikbadmus** [comment]
Great work @adebisi4145 , I've reviewed and ran several tests, and this works as intended.

**Comment by adebisi4145** [comment]
`Great work @adebisi4145 , I've reviewed and ran several tests, and this works as intended.`

Thank you @Malikbadmus for taking the time to review and test my code

**Pull request merged**